pinot-core/src/test/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionFactoryTest.java

```diff
@@ -444,6 +444,20 @@ public void testGetAggregationFunction() {
     assertEquals(aggregationFunction.getType(), AggregationFunctionType.PERCENTILETDIGESTMV);
     assertEquals(aggregationFunction.getColumnName(), "percentileTDigest95.0MV_column");
     assertEquals(aggregationFunction.getResultColumnName(), "percentiletdigestmv(column, 95.0)");
+
+    function = getFunction("bool_and");
```

Comment:
since we are adding this feature to V1 as well. we should add test for them in V1 too. see: https://github.com/apache/pinot/pull/9236 when adding COVAR_POP